bigbone/src/main/kotlin/social/bigbone/api/entity/StatusMention.kt

```diff
@@ -0,0 +1,33 @@
+package social.bigbone.api.entity
+
+import com.google.gson.annotations.SerializedName
+
+/**
+ * Mentions of users within the status content.
+ * @see <a href="https://docs.joinmastodon.org/entities/Status/#Mention">Mastodon API Status::Mention</a>
+ */
+data class StatusMention(
```

Comment:
Mastodon API has several of these cases, where a entity is defined only as a part of another. Wouldn't it make sense to use nested classes here, so that this becomes `Status.Mention` for example?

Reply:
Ah true, yes will change it. Thx! 😊